test/rubygems/test_gem_config_file.rb

```diff
@@ -19,7 +19,7 @@ def setup
     Gem::ConfigFile::PLATFORM_DEFAULTS.clear
 
     @env_gemrc = ENV['GEMRC']
-    ENV['GEMRC'] = ''
+    ENV['GEMRC'] = nil
```

Comment:
What's the behavior now with `ENV["GEMRC"] = ''`. I guess we could keep the current behaviour by checking for empty too in the `load_file` early return.

Reply:
Done :)